src/components/MovieCard.vue

```diff
@@ -1,6 +1,9 @@
 <template>
-  <div class="col-lg-2 col-md-2 col-sm-3 col-5 m-3 p-0">
-    <div class="card border-light">
+  <div
+    class="movie-card col-lg-2 col-md-2 col-sm-3 col-5 m-3 p-0"
+    @click="goToMoviePage"
```

Comment:
For better navigation, I suggest to add `tabindex="0"` for the root element. Also add event handlers for `@keydown.enter.esc="goToMoviePage"`. User will have possibility to navigate through the cards via keyboard